### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by radWorx**
Using Polar to test Lightning payments.

### CODE REVIEW

File: BTCPayServer.Tests/README.md [hunk 1]
```diff
@@ -26,6 +26,17 @@ You can also generate blocks:
 .\docker-bitcoin-generate.ps1 3
 ```
 
+### Using Polar to test Lightning payments
+
+-Install and run [Polar](https://lightningpolar.com/). Setup a small network of nodes.
```
Comment (britttttk) [inline BTCPayServer.Tests/README.md:31]:
```suggestion
- Install and run [Polar](https://lightningpolar.com/). Setup a small network of nodes.
```

File: BTCPayServer.Tests/README.md [hunk 2]
```diff
@@ -26,6 +26,17 @@ You can also generate blocks:
...
+-Goto your Store's General Settings and enable Lightning
```
Comment (britttttk) [inline BTCPayServer.Tests/README.md:32]:
```suggestion
- Goto your Store's General Settings and enable Lightning
```

File: BTCPayServer.Tests/README.md [hunk 3]
```diff
@@ -26,6 +26,17 @@ You can also generate blocks:
...
+-Build you connection string using the Connect infomation in the Polar app
```
Comment (britttttk) [inline BTCPayServer.Tests/README.md:33]:
```suggestion
- Build you connection string using the Connect information in the Polar app
```

**Review by britttttk — CHANGES_REQUESTED** [review]
Fixed typo and bullet point list. 

An alternative place for this info could be in [this document](https://github.com/btcpayserver/btcpayserver-doc/blob/master/docs/Contribute/ContributeDev/ContributeDevTest.md#testing-btcpay-server).

File: BTCPayServer.Tests/README.md [hunk 4]
```diff
@@ -26,6 +26,17 @@ You can also generate blocks:
...
+
+- Install and run [Polar](https://lightningpolar.com/). Setup a small network of nodes.
+- Goto your Store's General Settings and enable Lightning
+- Build you connection string using the Connect information in the Polar app
+
+LND Connection string example: 
+type=lnd-rest;server=https://127.0.0.1:8084/;macaroonfilepath="local path to admin.macaroon on your computer, without these quotes";allowinsecure=true
+
+Now you can create a lightning invoice on BTCPayserver regtest and make payment through polar
+PLEASE NOTE:
```
Comment (bolatovumar) [inline BTCPayServer.Tests/README.md:39]: I'm not sure what this `PLEASE NOTE:` is for here.

File: BTCPayServer.Tests/README.md [hunk 5]
```diff
@@ -26,6 +26,17 @@ You can also generate blocks:
 .\docker-bitcoin-generate.ps1 3
 ```
 
+### Using Polar to test Lightning payments
+
+- Install and run [Polar](https://lightningpolar.com/). Setup a small network of nodes.
+- Goto your Store's General Settings and enable Lightning
```
Comment (bolatovumar) [inline BTCPayServer.Tests/README.md:32]: `Goto` should be `Go to`

Comment (bolatovumar) [inline BTCPayServer.Tests/README.md:32]: Does `Store` need to be capitalized here?

File: BTCPayServer.Tests/README.md [hunk 6]
```diff
@@ -26,6 +26,17 @@ You can also generate blocks:
...
+- Build you connection string using the Connect information in the Polar app
+
+LND Connection string example: 
+type=lnd-rest;server=https://127.0.0.1:8084/;macaroonfilepath="local path to admin.macaroon on your computer, without these quotes";allowinsecure=true
+
+Now you can create a lightning invoice on BTCPayserver regtest and make payment through polar
```
Comment (bolatovumar) [inline BTCPayServer.Tests/README.md:38]: `polar` should be capitalized. Also missing a period at the end of the sentence.
Also, `and make payment` should be `and make a payment`.

File: BTCPayServer.Tests/README.md [hunk 7]
```diff
@@ -26,6 +26,17 @@ You can also generate blocks:
 .\docker-bitcoin-generate.ps1 3
 ```
 
+### Using Polar to test Lightning payments
+
+- Install and run [Polar](https://lightningpolar.com/). Setup a small network of nodes.
+- Goto your Store's General Settings and enable Lightning
+- Build you connection string using the Connect information in the Polar app
```
Comment (bolatovumar) [inline BTCPayServer.Tests/README.md:33]: `Build you` should probably be `Build your` here?

File: BTCPayServer.Tests/README.md (hunk 4 above)
Comment (radWorx) [reply BTCPayServer.Tests/README.md:39]: Was going to add a note about a break on exception. Will add and fix the recommendations.

File: BTCPayServer.Tests/README.md [hunk 8]
```diff
@@ -26,6 +26,21 @@ You can also generate blocks:
...
+### Using Polar to test Lightning payments
+
+- Install and run [Polar](https://lightningpolar.com/). Setup a small network of nodes.
+- Go to your store's General Settings and enable Lightning.
+- Build your connection string using the Connect information in the Polar app.
+
+LND Connection string example: 
+type=lnd-rest;server=https://127.0.0.1:8084/;macaroonfilepath="local path to admin.macaroon on your computer, without these quotes";allowinsecure=true
+
+Now you can create a lightning invoice on BTCPayserver regtest and make a payment through polar.
```
Comment (pavlenex) [inline BTCPayServer.Tests/README.md:38]:
```suggestion
Now you can create a lightning invoice on BTCPay Server regtest and make a payment through Polar.
```

**Review by dennisreimann — APPROVED** [review]
Tested, works as promised :)

You could also mention that ones needs to bring the Polar Bitcoin node to the same blockheight as the BTCPay node – otherwise the LN node shows up as not synched. On the other side this may not be necessary, as the LN connection message shows how many blocks difference there are and this is for advanced users anyways …

File: BTCPayServer.Tests/README.md [hunk 9]
```diff
@@ -26,6 +26,21 @@ You can also generate blocks:
...
+- Go to your store's General Settings and enable Lightning.
+- Build your connection string using the Connect information in the Polar app.
+
+LND Connection string example: 
+type=lnd-rest;server=https://127.0.0.1:8084/;macaroonfilepath="local path to admin.macaroon on your computer, without these quotes";allowinsecure=true
+
+Now you can create a lightning invoice on BTCPay Server regtest and make a payment through Polar.
+
+PLEASE NOTE: You may get an exception break in Visual Studio. You must quickly click "Continue" in VS so the invoice is generated.
+Or, uncheck the box that says, "Break when this exceptiontype is thrown".
```
Comment (bolatovumar) [inline BTCPayServer.Tests/README.md:41]: `exceptiontype` looks like there is a space missing?

**Comment by NicolasDorier** [comment]
Please move it after `Using the test lightning-cli` and mention that this is an alternative.

File: BTCPayServer.Tests/README.md (hunk 9 above)
Comment (radWorx) [reply BTCPayServer.Tests/README.md:41]: Yes. Will fix.

**Review by bolatovumar — APPROVED** [review]

**Pull request merged**